app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -276,6 +285,96 @@ def parse_namespaces(container_projects)
       parse_base_item(container_projects).except(:namespace)
     end
 
+
+    def parse_quota(resource_quota)
+      new_result = parse_base_item(resource_quota).except(:namespace)
+      new_result[:project] = @data_index.fetch_path(
+        :container_projects,
+        :by_name,
+        resource_quota.metadata["table"][:namespace])
+      new_result[:container_quota_items] = parse_quota_items resource_quota
+      new_result
+    end
+
+    def parse_quota_items(resource_quota)
+      new_result_h = Hash.new do |h, k|
+        h[k] = {
+            :resource       => k.to_s,
+            :quota_desired  => nil,
+            :quota_enforced => nil,
+            :quota_observed => nil
+        }
+      end
+
+      resource_quota.spec.hard.to_h.each do |resource_name, quota|
+        new_result_h[resource_name][:quota_desired] = quota
+      end
+
+      resource_quota.status.hard.to_h.each do |resource_name, quota|
+        new_result_h[resource_name][:quota_enforced] = quota
+      end
+
+      resource_quota.status.used.to_h.each do |resource_name, quota|
+        new_result_h[resource_name][:quota_observed] = quota
+      end
+
+      new_result_h.values
+    end
+
+    def parse_range(limit_range)
+      new_result = parse_base_item(limit_range).except(:namespace)
+      new_result[:project] = @data_index.fetch_path(
+        :container_projects,
+        :by_name,
+        limit_range.metadata["table"][:namespace])
+      new_result[:container_limit_items] = parse_range_items limit_range
+      new_result
+    end
+
+    def parse_range_items(limit_range)
+      new_result_h = create_limits_matrix
+
+      limit_range.spec.limits.each do |item|
+        item[:max].to_h.each do |resource_name, limit|
+          new_result_h[item[:type].to_sym][resource_name.to_sym][:max] = limit
+        end
+
+        item[:min].to_h.each do |resource_name, limit|
+          new_result_h[item[:type].to_sym][resource_name.to_sym][:min] = limit
+        end
+
+        item[:default].to_h.each do |resource_name, limit|
+          new_result_h[item[:type].to_sym][resource_name.to_sym][:default] = limit
+        end
+
+        item[:defaultRequest].to_h.each do |resource_name, limit|
+          new_result_h[item[:type].to_sym][resource_name.to_sym][:default_request] = limit
+        end
+
+        item[:maxLimitRequestRatio].to_h.each do |resource_name, limit|
+          new_result_h[item[:type].to_sym][resource_name.to_sym][:max_limit_request_ratio] = limit
+        end
+      end
+      new_result_h.values.collect(&:values).flatten
+    end
```

Comment:
@simon3z Not sure how to get the new parse_range_items above to pass rubocop without losing readability.